tests/model_registry/async_job/conftest.py

```diff
@@ -114,6 +114,39 @@ def oci_secret_for_async_job(
         yield secret
 
 
+@pytest.fixture(scope="class")
+def async_upload_image(admin_client: DynamicClient) -> str:
+    """
+    Get the async upload image dynamically from the model-registry-operator-parameters ConfigMap.
+
+    This fetches the image from the cluster at runtime instead of using a hardcoded value.
+
+    Args:
+        admin_client: Kubernetes client for resource access
+
+    Returns:
+        str: The async upload image URL from the ConfigMap
+
+    Raises:
+        KeyError: If the ConfigMap or the required key doesn't exist
+    """
+    config_map = ConfigMap(
+        client=admin_client,
+        name="model-registry-operator-parameters",
```

Comment:
Is this going to be same for odh nightlies? Should the tests be marked as downstream only?